sqlmesh/dbt/model.py

```diff
@@ -76,6 +76,10 @@ class ModelConfig(BaseModelConfig):
     # redshift
     bind: t.Optional[bool] = None
 
+    # Private fields
+    _sql_config: t.Optional[SqlStr] = None
```

Comment:
`_sql_with_config`? or maybe just `_sql`?

Reply:
Now called sql_embedded_config (its only the config portion)